Example/Backpack/SwiftUI/Components/Calendar/CalendarExampleSingleView.swift

```diff
@@ -49,28 +49,39 @@ struct CalendarExampleSingleView: View {
             self.monthScroll = .init(monthToScroll: date, animated: true)
         }
 
-        _selectedDate = State(initialValue: date)
+        _selection = State(initialValue: .single(date))
     }
     
     var body: some View {
         VStack {
             HStack {
                 BPKText("Selected date:", style: .caption)
-                if let selectedDate {
-                    BPKText("\(formatter.string(from: selectedDate))", style: .caption)
+                if case .single(let date) = selection, let date {
+                    BPKText("\(formatter.string(from: date))", style: .caption)
+                } else if case .wholeMonth(let month) = selection {
+                    BPKText("\(formatter.string(from: month.lowerBound))", style: .caption)
                 }
             }
             BPKCalendar(
                 selectionType: .single(
-                    selected: $selectedDate,
+                    selected: $selection,
                     accessibilityConfigurations: SingleAccessibilityConfigurations(
-                        selectionHint: "Double tap to select date"
+                        selectionHint: "Double tap to select date",
+                        wholeMonth: nil
                     )
                 ),
                 calendar: calendar,
                 validRange: validRange,
                 initialMonthScroll: monthScroll
             )
+            .monthAccessoryAction { _ in
+                return CalendarMonthAccessoryAction(
```

Comment:
I wonder if the accessibility setting for this should be here instead, to avoid being possible to send nil to that, when you do have an action. wdyt?

Reply:
Good point. 👍

I made this config as part whole month selection state. So if the consumer doesn't want to use the wholeMonth they don't have to provide this value. Removed this optional. ✅